src/content/guides/integrating-third-party-asset.mdx

```diff
@@ -108,6 +108,29 @@ curl https://auth.centrapay.com/api/.well-known/jwks.json
 }
 ```
 
+To verify a request start by decripting the JWT.
```

Comment:
@laurencemcknight Typo here - "decripting"

Reply:
Decrypting is not the right word. Decoding is more accurate

Reply:
Hmm, I think it's more decrypting. We encrypt the message with a specific private key and then they need the public key in order to decrypt the message right?

Reply:
I think encoding/decoding seems more like applying an algorithm without any private/public keys.

Reply:
@daallen303 "Encoding" and "decoding" is the common terminology online used for JWTs. Additionally we already use that terminology in our docs for JWTs.